backend/.env

```
@@ -1,3 +1,4 @@
 DATABASE_MONGODB_URL: 'mongodb://localhost:27017/coderscamp'
-
+ACCESS_TOKEN: 'dwadwadawgafawdawdawdaw'
```

```suggestion
ACCESS_TOKEN: 'EXAMPLE_ACCESS_TOKEN'
```
albo jakąś inną nazwę :)